packages/office-ui-fabric-react/src/components/Layer/Layer.types.ts

```diff
@@ -55,6 +55,13 @@ export interface ILayerProps extends React.HTMLAttributes<HTMLDivElement | Layer
    * default while allowing users to opt in to the new event bubbling functionality.
    */
   eventBubblingEnabled?: boolean;
+
+  /**
```

Comment:
Perhaps this should be `insertMode: 'append' | 'prepend'`, where `'append'` is the default.

Reply:
I don't have a strong preference, but if we go the `'append' | 'prepend'` approach we have to use defaultProps which I'm never a fan of if it can be avoided (with a bool where undefined is falsy, for example)

Reply:
@ThomasMichon , any strong preference here? I'd like to get this merged, but want to make sure you agree :)

Reply:
@ThomasMichon I'm going to merge this today unless you have anything to add